website/pages/blog/migrating-from-v0-to-v1.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Migrating from CloudQuery v0 to v1
+tag: announcement
+date: 2022/10/03
+description: >-
+  A guide for users migrating from CloudQuery v0 to v1
+author: hermanschaaf
+---
+
+import { BlogHeader } from "../../components/BlogHeader"
+
+<BlogHeader/>
+
+We are thrilled to announce the release of the first major version of CloudQuery--see our [v1 announcement blog post](/blog/cloudquery-v1-release) for details! With the new release comes a range of new exciting features, and this page is here to help you migrate an existing CloudQuery installation from v0 to v1.
+
+## Changes in V1
+
+The [announcement blog post](/blog/cloudquery-v1-release) lists many of the important improvements, and we won't re-iterate them all here. Most changes are internal and developer-facing, but some do impact existing CloudQuery teams. Those are:
+- [changes to CLI commands](#changes-to-the-cli-commands)
+- [changes to the configuration format](#changes-to-the-configuration-format)
+- [changes to tables and schemas](#changes-to-tables-and-schemas)
+
+## Changes to the CLI Commands
+
+Users of CloudQuery v0 would be familiar with the main commands `init` and `fetch`, as well as some others. These are no longer in V1, but similar commands have replaced them to reflect that CloudQuery now supports both source and destination plugins.
+
+### Generate commands
+
+- Use `cloudquery gen source <plugin-name>` to generate the config for a source plugin (e.g. `cloudquery gen source aws`)
+- Use `cloudquery gen destination <plugin-name>` to generate the config for a destination plugin (e.g. `cloudquery gen destination postgresql`)
+
+`cloudquery init` has been replaced by `cloudquery gen` and is no longer part of the CLI. See `cloudquery gen --help` for more information, or check our [online reference](/docs/reference/cli/cloudquery_generate).
+
+### Sync commands
+
+`cloudquery sync` replaces the v0 `cloudquery fetch` command.
+
+Functionally it is still the same: it loads data from a source to a destination, but `sync` now supports multiple destinations, while `fetch` only supported PostgreSQL. With this change also comes a change in expected config format, see the [next section](#changes-to-the-configuration-format) for more details on this.
+
+`cloudquery sync` needs to be passed a path to a config file or directory containing config files. So for example, to sync using a single yaml file:
+
+```
+cloudquery sync config.yml
```

Comment:
The guide looks great, but I'm curious how this works. Don't we need at least 2 configuration files (1 for a source and 1 for a destination)?

Reply:
Yeah, it's explained a little bit lower down in the post, but probably should be mentioned here as well: you will need to separate the different parts of the config with `---`